include/picongpu/plugins/particleMerging/VoronoiCell.hpp

```diff
@@ -134,7 +134,7 @@ namespace particleMerging
             const float_X weighting
         )
         {
-            nvidia::atomicAllInc( acc, &this->numMacroParticles, ::alpaka::hierarchy::Threads{} );
+            cupla::atomicAdd(acc, &this->numMacroParticles, static_cast<uint32_t>(1), ::alpaka::hierarchy::Threads{} );
```

Comment:
Note: here different threads in a block may or may not have the same address. So an atomic operation is needed, but `atomicAdd`, not `atomicAllInc`.